data/CT/incentives.json

```diff
@@ -523,5 +523,145 @@
+    ],
+    "item": "new_electric_vehicle",
+    "program": "ct_electricVehicleAndChargingRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate up to $1,500 for purchasing a new EV (model year 2020 or newer). Can be combined with federal, state, or manufacturer incentives."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "CT-30",
+    "authority_type": "utility",
+    "authority": "ct-norwich-public-utilities",
+    "type": "rebate",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "used_electric_vehicle",
+    "program": "ct_electricVehicleAndChargingRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate up to $1,000 for purchasing a used EV (model year 2019 or newer). Can be combined with federal, state, or manufacturer incentives."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "CT-31",
+    "authority_type": "utility",
+    "authority": "ct-norwich-public-utilities",
+    "type": "rebate",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "electric_vehicle_charger",
+    "program": "ct_electricVehicleAndChargingRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 1000
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1,000 rebate for purchasing a residential use Level 2 EVSE. Must apply within 60 days of installation."
+    },
+    "start_date": 2024,
+    "end_date": 2024
+  },
+  {
+    "id": "CT-32",
+    "authority_type": "state",
+    "authority": "ct-deep",
+    "type": "pos_rebate",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "new_electric_vehicle",
+    "program": "ct_cheapr",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2250
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$2,250 rebate for a qualifying new electric vehicle."
+    }
+  },
+  {
+    "id": "CT-33",
+    "authority_type": "state",
+    "authority": "ct-deep",
+    "type": "pos_rebate",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "new_electric_vehicle",
+    "program": "ct_cheapr",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 2000
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "low_income": "ct-cheapr",
+    "short_description": {
+      "en": "Bonus $2,000 off a qualifying new electric vehicle for income-eligible individuals."
+    }
+  },
+  {
+    "id": "CT-34",
+    "authority_type": "state",
+    "authority": "ct-deep",
+    "type": "pos_rebate",
+    "payment_methods": [
+      "pos_rebate",
+      "rebate"
```

Comment:
Does this program actually have two ways of paying out the incentive to users (a POS rebate and a later on rebate?)

Reply:
The way I read it was that you could either get the POS rebate or do a mail-in rebate instead.